UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -225,6 +225,15 @@ public bool HasAccess(Access access)
 		return accessSyncList.Contains((int) access);
 	}
 
+	public bool HasAccess(List<Access> access)
```

Comment:
Nitpick, but could the method name more clearly state what this is actually checking? Something like `HasAnyAccess`?

Reply:
Why? It's more confusing having differently named functions. Having one function that can have different arguments but is expected to return the same result makes more sense